src/plugin/impl/s3-copy-build-task-plugin.ts

```diff
@@ -124,15 +131,21 @@ export class CopyToS3TaskPlugin implements IBuildTaskPlugin<IS3CopyBuildTaskConf
         await s3client.putObject(request).promise();
     }
 
-    async createBody(task: IS3CopyTask): Promise<S3.Body> {
+    private async createBody(task: IS3CopyTask): Promise<S3.Body> {
         if (!task.zipBeforePut) {
-            return fs.readFileSync(task.localPath);
+            const fileContent = fs.readFileSync(task.localPath);
+            const debugFilename = `${task.localPath}.debug`;
```

Comment:
instead of just appending `.debug` it might be better to use path.basename(task.localPath). this ensures that a relative path is used and the nunjucks debug files are written to the right folder